### PR TITLE
Update TIP 4.5

### PR DESCRIPTION
**Pull request by itgoldio**
Add license localization.
The implementation of the standard methods has been removed.
Removed the binding of storing licenses in a specific repository.

### CODE REVIEW

**Comment by rualekseev** [comment]
Do you think that getLicenseName it's an unnecessary parameter?

**Comment by rualekseev** [comment]
Can license change?
we have 3 options

- Don't detail it. Developer can do anything with license

- Forbid license change after nft deploy

- Allow license change after nft deploy and add event to standart

**Review by ilyar — DISMISSED** [review]
The proposed change is incompatible

**Comment by itgoldio** [comment]
Pull request was updated. 

Removed the way to work with different localizations.

The following changes are currently being proposed:
1. do not fix the license storage location at Can’tBeEvil repository
2. methods implementation removed

**Review by vp-mazekine — CHANGES_REQUESTED** [review]
Thank you for the update. However, it still makes existing collections incompatible:
1. The proposed update changes the name and response type of function: from `getLicenseURI() returns (string)` to `getLicenceInfo() returns (LicenceInfo)` (line 38 of the updated file), which breaks the support of the existing collections
2. It also removes the support of the interface CantBeEvil which initializes the license within an NFT and doesn't propose any other method of initializing
3. The `TIP-6.1` identifier is specified incorrectly (please refer to the corresponding standard to learn how to derive it from the interface)
4. The interface ITIP4_5 doesn't include the proposed functions `getLicenceInfo` and `getLicenceURIByLocale`

Moreover, a few minor points to be addressed:
1. Use `pragma ever-solidity` instead of `pragma ton-solidity.` Despite the fact that they are both legitimate, we aim to get rid of the FreeTON legacy in favor of Evercale and fix all related mentions wherever possible.
2. As a rule of thumb, use American (International) English instead of British: `license` instead of `licence`.

To sum up, I still don't see any necessity to make breaking changes to the existing standard in use instead of proposing an alternative one that will be its inheritor (like ERC-777 to ERC-20). I definitely recommend you introduce a new standard.

**Comment by itgoldio** [comment]
Thanks for your suggestions!

1. fixed
2. CantBeEvil is listed as one of the sources, there is a direct link to the repository in the text
3. fixed. In the previous version of the documentation, the wrong identifier was specified, the correct one is: 0x8A1EB91
4. fixed

Minor points:
1. fixed
2. fixed

The proposed changes are compatible with previous versions and are therefore proposed within the scope of this standard.

**Review by vp-mazekine — CHANGES_REQUESTED** [review]
Thank you, it has become better.

However, please make the last fixes:
1. Return the "Motivation" section to justify the reasoning for the appearance of this standard.
2. Leave the historical reference to the a16z's "Can't Be Evil" licensing standard, as it is vital for network public relations.
3. Amend the abstract in line with the standard update.
4. Mention two possible interface identifiers: 0x1E4848D4 for methods without the `responsible` modifier and 0x8A1EB91 otherwise.
5. Refer to standard TIP-6.1 when mentioning interface identifier, as self-reference makes no sense.
6. Write NFT in capital on line 17
7. Please don't downgrade the compiler version the was initially specified in the standard (0.61.2)

Thank you for your contribution!

**Review by vp-mazekine — APPROVED** [review]

**Review by ilyar — APPROVED** [review]

**Review by nerzh — APPROVED** [review]

**Review by ilyar — CHANGES_REQUESTED** [review]
please follow https://github.com/everscale-org/docs/blob/main/CONTRIBUTING.md

**Review by ilyar — APPROVED** [review]
